### ISSUE TITLE
Keyboard navigation between linter messages fails if there are no messages in the current file

### ISSUE THREAD

**Issue by bobsilverberg** [mozilla/addons-code-manager#837]
### Describe the problem and steps to reproduce it:

When you click a key for linter message navigation while looking at a file without any linter messages, the navigation fails.

STR:

1. Load a version that has linter messages for some of the files.
2. Select a file without any linter messages.
3. Click `z` to go to the next message.

### What happened?

No navigation occurs and a console message appears saying "Error: No messages found for current path: {filename}".

### What did you expect to happen?

The next file with a linter message should load, and the line with the message should be selected.

**Comment by ioanarusiczki** [comment]
@bobsilverberg I am positioned on a file without linter messages and try `z` or `a` the next/previous file with a linter message is loaded and the line with the message selected.

![not reproducible](https://user-images.githubusercontent.com/33448286/59857230-31f95980-9381-11e9-99eb-9b0cd676779b.gif)

Verified on dev - FF67(Win10)
